tests/Fixtures/ChoiceFactoryMock.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Tests\Fixtures;
+
+use WMDE\Fundraising\Frontend\BucketTesting\FeatureToggle;
+use WMDE\Fundraising\Frontend\Factories\UnknownChoiceDefinition;
+
+/**
+ * @license GNU GPL v2+
+ */
+class ChoiceFactoryMock {
```

Comment:
In our test fixture code naming we try to be more precise what the code does, see https://www.entropywins.wtf/blog/2016/05/13/5-ways-to-write-better-mocks/ for the definitions of the different kinds of test doubles. In this case, I'd call the class `FakeChoiceFactory`.